phoenix-core/src/main/java/org/apache/phoenix/schema/PTable.java

```diff
@@ -583,6 +583,7 @@ private static int getReservedQualifier(byte[] bytes, int offset, int length) {
     PName getName();
     PName getSchemaName();
     PName getTableName();
+    PName getPhysicalTableNameColumnInSyscat();
```

Comment:
Thinking about this more, I don't think just changing the name solves the problem of having two methods that are almost-but-not-quite the same thing. I'd still be confused about which to call. 

Can we not use the existing getPhysicalNames() to return the physical table name column from syscat in the situations where that's appropriate?

Reply:
getPhysicalName already returns physical table name column from syscat when appropriate.

We still need 2 methods. One represents the actual column in Syscat. The other is inferred (like views).